<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>997</width>
    <height>577</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QToolBox" name="toolBoxMain">
      <widget class="QWidget" name="pageCalc">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>979</width>
         <height>404</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Calc</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="CalcFrame" name="frameCalc">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pageMilkReception">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>100</width>
         <height>30</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Сдача молока</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="MilkReceptionAddFrame" name="frameMilkReceptionAdd">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pageEditMilkReception">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>100</width>
         <height>30</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Редактировать сдачи молока</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="FrameEditMilkReception" name="frameEditReceptionMilk">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>997</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDb">
    <property name="title">
     <string>Бд</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuLocalities">
    <property name="title">
     <string>Населенные пункты</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuMilkPoints">
    <property name="title">
     <string>Молокопункты</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuDeliverers">
    <property name="title">
     <string>Сдатчики</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Файл</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Справка</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuDb"/>
   <addaction name="menuLocalities"/>
   <addaction name="menuDeliverers"/>
   <addaction name="menuMilkPoints"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="toolBarDb">
   <property name="windowTitle">
    <string>База данных</string>
   </property>
   <property name="toolTip">
    <string>База данных</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="toolBarLocalities">
   <property name="windowTitle">
    <string>Населенные пункты</string>
   </property>
   <property name="toolTip">
    <string>Населенные пункты</string>
   </property>
   <property name="statusTip">
    <string>Населенные пункты</string>
   </property>
   <property name="whatsThis">
    <string>Населенные пункты</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="toolBarDeliverers">
   <property name="windowTitle">
    <string>Сдатчики</string>
   </property>
   <property name="toolTip">
    <string>Сдатчики</string>
   </property>
   <property name="statusTip">
    <string>Сдатчики</string>
   </property>
   <property name="whatsThis">
    <string>Сдатчики</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="toolBarMilkPoints">
   <property name="windowTitle">
    <string>Молокопункты</string>
   </property>
   <property name="toolTip">
    <string>Молокопункты</string>
   </property>
   <property name="statusTip">
    <string>Молокопункты</string>
   </property>
   <property name="whatsThis">
    <string>Молокопункты</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionCreateDb">
   <property name="icon">
    <iconset resource="../../res.qrc">
     <normaloff>:/img/db/db_create.png</normaloff>:/img/db/db_create.png</iconset>
   </property>
   <property name="text">
    <string>Создать бд</string>
   </property>
   <property name="toolTip">
    <string>Создать бд</string>
   </property>
  </action>
  <action name="actionOpenDb">
   <property name="icon">
    <iconset resource="../../res.qrc">
     <normaloff>:/img/db/db_open.png</normaloff>:/img/db/db_open.png</iconset>
   </property>
   <property name="text">
    <string>Открыть бд</string>
   </property>
   <property name="toolTip">
    <string>Открыть бд</string>
   </property>
  </action>
  <action name="actionCloneDb">
   <property name="icon">
    <iconset resource="../../res.qrc">
     <normaloff>:/img/db/db_clone.png</normaloff>:/img/db/db_clone.png</iconset>
   </property>
   <property name="text">
    <string>Клонировать бд</string>
   </property>
   <property name="toolTip">
    <string>Клонировать бд</string>
   </property>
  </action>
  <action name="actionLocalityAdd">
   <property name="icon">
    <iconset resource="../../res.qrc">
     <normaloff>:/img/locality/locality_add.png</normaloff>:/img/locality/locality_add.png</iconset>
   </property>
   <property name="text">
    <string>Добавить населенный пункт</string>
   </property>
   <property name="toolTip">
    <string>Добавить населенный пункт</string>
   </property>
  </action>
  <action name="actionLocalityEdit">
   <property name="icon">
    <iconset resource="../../res.qrc">
     <normaloff>:/img/locality/locality_edit.png</normaloff>:/img/locality/locality_edit.png</iconset>
   </property>
   <property name="text">
    <string>Редактировать населенный пункт</string>
   </property>
   <property name="toolTip">
    <string>Редактировать населенный пункт</string>
   </property>
  </action>
  <action name="actionLocalityDel">
   <property name="icon">
    <iconset resource="../../res.qrc">
     <normaloff>:/img/locality/locality_remove.png</normaloff>:/img/locality/locality_remove.png</iconset>
   </property>
   <property name="text">
    <string>Удалить населенный пункт</string>
   </property>
   <property name="toolTip">
    <string>Удалить населенный пункт</string>
   </property>
  </action>
  <action name="actionDelivererAdd">
   <property name="icon">
    <iconset resource="../../res.qrc">
     <normaloff>:/img/user/user_add.png</normaloff>:/img/user/user_add.png</iconset>
   </property>
   <property name="text">
    <string>Добавить сдатчика</string>
   </property>
   <property name="toolTip">
    <string>Добавить сдатчика</string>
   </property>
  </action>
  <action name="actionDelivererEdit">
   <property name="icon">
    <iconset resource="../../res.qrc">
     <normaloff>:/img/user/user_edit.png</normaloff>:/img/user/user_edit.png</iconset>
   </property>
   <property name="text">
    <string>Редактировать сдатчика</string>
   </property>
   <property name="toolTip">
    <string>Редактировать сдатчика</string>
   </property>
  </action>
  <action name="actionDelivererDel">
   <property name="icon">
    <iconset resource="../../res.qrc">
     <normaloff>:/img/user/user_remove.png</normaloff>:/img/user/user_remove.png</iconset>
   </property>
   <property name="text">
    <string>Удалить сдатчика</string>
   </property>
   <property name="toolTip">
    <string>Удалить сдатчика</string>
   </property>
  </action>
  <action name="actionMilkPointAdd">
   <property name="icon">
    <iconset resource="../../res.qrc">
     <normaloff>:/img/milk_point/milk_point_add.png</normaloff>:/img/milk_point/milk_point_add.png</iconset>
   </property>
   <property name="text">
    <string>Добавить молокопункт</string>
   </property>
   <property name="toolTip">
    <string>Добавить молокопункт</string>
   </property>
  </action>
  <action name="actionMilkPointEdit">
   <property name="icon">
    <iconset resource="../../res.qrc">
     <normaloff>:/img/milk_point/milk_point_edit.png</normaloff>:/img/milk_point/milk_point_edit.png</iconset>
   </property>
   <property name="text">
    <string>Редактировать молокопункт</string>
   </property>
   <property name="toolTip">
    <string>Редактировать молокопункт</string>
   </property>
  </action>
  <action name="actionMilkPointDel">
   <property name="icon">
    <iconset resource="../../res.qrc">
     <normaloff>:/img/milk_point/milk_point_remove.png</normaloff>:/img/milk_point/milk_point_remove.png</iconset>
   </property>
   <property name="text">
    <string>Удалить молокопункт</string>
   </property>
   <property name="toolTip">
    <string>Удалить молокопункт</string>
   </property>
  </action>
  <action name="actionRemoveLocalities">
   <property name="text">
    <string>Удалить все населенные пункты</string>
   </property>
   <property name="toolTip">
    <string>Удалить все населенные пункты</string>
   </property>
  </action>
  <action name="actionRemoveDeliverers">
   <property name="text">
    <string>Удалить всех сдатчиков</string>
   </property>
   <property name="toolTip">
    <string>Удалить всех сдатчиков</string>
   </property>
  </action>
  <action name="actionRemoveAllMilkReception">
   <property name="text">
    <string>Удалить все сдачи молока</string>
   </property>
   <property name="toolTip">
    <string>Удалить все сдачи молока</string>
   </property>
  </action>
  <action name="actionRemoveMilkPoints">
   <property name="text">
    <string>Удалить все молокопункты</string>
   </property>
   <property name="toolTip">
    <string>Удалить все молокопункты</string>
   </property>
  </action>
  <action name="actionShowDialogSettings">
   <property name="icon">
    <iconset resource="../../res.qrc">
     <normaloff>:/img/setup.png</normaloff>:/img/setup.png</iconset>
   </property>
   <property name="text">
    <string>Настройки</string>
   </property>
   <property name="toolTip">
    <string>Отобразить диалог настроек</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Выход</string>
   </property>
   <property name="toolTip">
    <string>Выход из программы</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="text">
    <string>Версия Qt</string>
   </property>
   <property name="toolTip">
    <string>Информация о используемой версии Qt</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="../../res.qrc">
     <normaloff>:/img/info.png</normaloff>:/img/info.png</iconset>
   </property>
   <property name="text">
    <string>Иформация о программе</string>
   </property>
   <property name="toolTip">
    <string>Иформация о программе</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MilkReceptionAddFrame</class>
   <extends>QFrame</extends>
   <header location="global">MilkReceptionAddFrame.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CalcFrame</class>
   <extends>QFrame</extends>
   <header location="global">CalcFrame.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FrameEditMilkReception</class>
   <extends>QFrame</extends>
   <header location="global">FrameEditMilkReception.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../res.qrc"/>
 </resources>
 <connections/>
</ui>
